### PR TITLE
pull keys into .env, confirm delete, etc

### LINKED ISSUE #2: Add delete link/confirmation & service integration

### LINKED ISSUE #10: Abstract out API / service keys to .env vars
Fastly key, dictionary & service config IDs -- these should all be in the `.env` file.

### PR DESCRIPTION
**Pull request by katiecrane**
- pulls fastly keys into .env
- users must confirm to delete redirects
- added some light validation
- force paths to begin with `/` if they don't already

Issues:
Fixes #2 
Fixes #10

### CODE REVIEW

File: dist/main.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```
Comment (DFurnes) [inline dist/main.js]: This file should be git-ignored!

File: package.json [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
+  "scripts": {
+    "start": "webpack",
+    "build": "NODE_ENV=production webpack"
   },
   "dependencies": {
-    "laravel-elixir": "^4.0.0",
-    "bootstrap-sass": "^3.0.0",
     "@dosomething/forge": "^6.6.2",
-    "dosomething-modal": "^0.3.0"
+    "bootstrap-sass": "^3.0.0",
```
Comment (DFurnes) [inline package.json]: This can probably be removed.

File: dist/main.js (hunk 1 above)
Comment (DFurnes) [inline dist/main.js]: Oh, actually deleted all-together. I think this is from when we were messing around with the output paths.

**Comment by DFurnes** [comment]
The `public/dist` folder should be gitignored, and we can set up Wercker to build those before deploying.

**Comment by DFurnes** [comment]
Otherwise, looks good!

File: README.md [hunk 3]
```diff
@@ -1,2 +0,0 @@
-# nightwing
-A redirect management application in Laravel.
```
Comment (DFurnes) [inline README.md:2]: :skull: No more readme?

File: app/Http/Controllers/RedirectsController.php [hunk 4]
```diff
@@ -42,7 +42,17 @@ public function create()
...
     {
-        $redirect = Redirect::create($request->all());
+        $this->validate($request, [
+            'path' => 'required',
+            'target' => 'required',
+        ]);
+
+        $request = $request->all();
+        if(substr($request['path'], 0, 1) !== '/')
+        {
```
Comment (DFurnes) [inline app/Http/Controllers/RedirectsController.php]: This bracket should be on the same line as the `if`, and no-space between `if` and `(...)`! :space_invader:

Comment (DFurnes) [inline app/Http/Controllers/RedirectsController.php]: It might be a good idea to set up StyleCI on this repo before wrapping everything up.

**Comment by DFurnes** [comment]
Looks good! One tiny style nitpick, and good to merge!

File: README.md (hunk 3 above)
Comment (katiecrane) [inline README.md:2]: oops

**Comment by DFurnes** [comment]
We should also make sure to invalidate all these keys after merging this! :boom: :bomb: :lock_with_ink_pen:

**Comment by DFurnes** [comment]
Hmm, capitalization on the readme changed. No biggie, but strange! :ship:

**Pull request merged**